test/fixtures/tape-standalone.js

```diff
@@ -0,0 +1,39 @@
+const test = require("tape-async");
+
+const DEFAULT_TIMEOUT = 500;
+
+let browser = "unknown";
+if (navigator.userAgent.includes("Chrome/")) {
+  browser = "Chrome";
+} else if (navigator.userAgent.includes("Firefox/")) {
+  browser = "Firefox";
+}
+
+// Export as a global a wrapped test function which enforces a timeout by default.
+window.test = (desc, fn) => {
```

Comment:
To make it clear that `test` is not a regular tap `test`, use a different name, e.g. `describeBrowserTest`.

Reply:
uhm... not sure that it is worth the additional verbosity in the test files, in the end it is just a wrapper around tape which ensure that the test will timeout (because it wasn't and I would really prefer that it does timeout at some point if no message is being received, so that we can more easily guess where the test was failing directly from the failures on travis).

Reply:
If it was a real `test` from tap, then `test.skip` would also work. But we don't use it yet, so if you prefer, we can keep `test`.